Package.swift

```diff
@@ -72,21 +78,31 @@ extension Target {
     static var coreTests: Target {
         .testTarget(
             name: "BowLiteCoreTests",
-            dependencies: [.target(name: Target.core.name),
-                           .target(name: Target.bowLiteLaws.name)])
+            dependencies: [
+                .target(name: Target.core.name),
+                .target(name: Target.laws.name)
+            ]
+        )
     }
 
     static var effectsTests: Target {
-        .testTarget(name: "BowLiteEffectsTests",
-                    dependencies: [.target(name: Target.effects.name),
-                                   .target(name: Target.bowLiteLaws.name)])
+        .testTarget(
+            name: "BowLiteEffectsTests",
+            dependencies: [
+                .target(name: Target.effects.name),
+                .target(name: Target.laws.name)
+            ]
+        )
     }
 
-    static var bowLiteLaws: Target {
-        .testTarget(
+    static var laws: Target {
+        .target(
```

Comment:
I think we should test it in nef-playgrounds

Reply:
It does not work on iPad, because it has the `SwiftCheck` dependency.

Reply:
This target is not exposed to the outside, so it won't cause any troubles.

Reply:
As `swift packages` describe works, it is a problem for us, or unless we will need to update nef-playgroundbook. We are using the whole `targets` to transform them into modules. Anyway, we can try to use `type` of each target to know if it is a  test or a library; but we need to see how this change affect to Bow.